src/test/java/com/google/sps/UserTest.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.Math;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.SmallCityService;
+import com.google.sps.data.Listing;
+import com.google.sps.data.MapLocation;
+import com.google.sps.data.User;
+
+/** Test class for User Object **/
+@RunWith(JUnit4.class)
+public final class UserTest {
+  // Pittsburgh Google Office Location
+  private final MapLocation USER_MAP_LOCATION = new MapLocation(40.45717, -79.91669);
+  private final String USER_ZIPCODE = "15206";
+  private final String USER_CITY = "Pittsburgh";
+  private final String USER_ADDRESS = "6425 Penn Ave, Pittsburgh, PA 15206";
+  
+  private User userWithMapLocation;
+  private User userWithZipCode;
+  private User userWithCity;
+  private User userWithAddress;
```

Comment:
nit: Typically we prefix these with "TEST_" and "test_" respectively

Reply:
just changed.